pyproject.toml

```diff
@@ -20,7 +20,7 @@ dependencies = [
     # If you update the minimum required jsonschema version, also update it in build.yml
     "jsonschema>=3.0",
     "packaging",
-    "narwhals>=1.26.0"
```

Comment:
is it necessary to bump the minimum dependency if the changes only affect internal Altair type checking?

Reply:
thanks for explaining!

true that users wouldn't interact with it directly, but we are seeing more and more packages using it, and hopefully none of them are setting an upper bound on their narwhals requirement, but there is a risk that they do that, and bumping too much risks that users end up with "dependency hell"

agree that 1.26 -> 1.27.1 isn't a big jump anyway 😄 just a comment for the future